CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+## Parity-Ethereum [v2.7.1](https://github.com/paritytech/parity-ethereum/releases/tag/v2.7.1)
+Parity Ethereum v2.7.2-stable is a patch version release of parity-ethereum. 
+Starting in the 2.7.x series of releases, parity-ethereum is switching to a single `stable` release
+track. As a result, any clients that currently receive updates from the `beta`
+track should switch to the `stable` track.
+Due to database format changes, upgrading from 2.5.x or 2.6.x is one-way only. 
+
+The full list of included changes:
```

Comment:
I guess #11455 doesn't add any value in the changelog, right?

Reply:
added to the list (I was hoping it won't trigger the ci as there is it doesn't touch any rust file)